pageserver/src/tenant/remote_timeline_client/download.rs

```diff
@@ -365,7 +362,39 @@ async fn do_download_index_part(
         &format!("download {remote_path:?}"),
         cancel,
     )
-    .await?;
+    .await
+}
+
+pub async fn do_download_tenant_manifest(
+    storage: &GenericRemoteStorage,
+    tenant_shard_id: &TenantShardId,
+    cancel: &CancellationToken,
+) -> Result<(TenantManifest, Generation), DownloadError> {
+    // TODO: generation support
+    let generation = Generation::none();
```

Comment:
It would be nice to use `Generation::new(1)` here instead, so that we always have generations in the path from day 0 (Generation::none only exists for backward compat to pre-generation indices).  That way any code that cares about generations never has to handle the None case.

Reply:
done